lib/rubocop/cop/cop.rb

```diff
@@ -147,8 +147,7 @@ def duplicate_location?(location)
       end
 
       def correct(node)
-        reason = reason_to_not_correct(node)
-        return reason if reason
+        return if reason_to_not_correct(node)
```

Comment:
This is for preserving class line-length of `182`: https://github.com/rubocop-hq/rubocop/blob/e97eb5ce74b3319c21de05f6c1fd78493bd55b47/.rubocop_todo.yml#L15-L16

Reply:
This changes the return value and the status of the offense in turn. It is strange that there is no test coverage for this.

Reply:
Ha! my bad.